setup.py

```diff
@@ -32,8 +35,6 @@ def get_version(rel_path):
     version=get_version("blackjax/__init__.py"),
     description="Flexible and fast inference in Python",
     packages=setuptools.find_packages(),
-    install_requires=[
-        "jax==0.2.7",
-        "jaxlib==0.1.57",
-    ],
+    install_requires=INSTALL_REQS
```

Comment:
What is the advantage of having the requirements outside setup.py? I'm asking because this is the first time I see this.

Reply:
so you can run `pip install -r requirements.txt`, or so automated bots like the github bot can scan the requirements.txt